protocol/jsm/v2/receiver.go

```diff
@@ -0,0 +1,169 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package jsm
+
+import (
+	"context"
+	"github.com/nats-io/jsm.go"
```

Comment:
for end user app its best to use just nats.go and it's `nc.JetStream()` context, those are designed for end user things - jsm.go was built during our early tech preview phases to explore the api design and what we want, but now mainly there to support the CLI.

Reply:
@ripienaar updated